digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/NodesAndJunctionsService.scala

```diff
@@ -141,6 +141,22 @@ class NodesAndJunctionsService(roadwayDAO: RoadwayDAO, roadwayPointDAO: RoadwayP
     }
   }
 
+  /**
+    * Check if any of the junction points is on a reserved road part (VIITE-2518).
```

Comment:
Could be elaborated:
"A junction point belonging to a reserved road part shall not be edited/editable."

Reply:
Se ei kuulu tän funktion skooppiin. Tää funktio tekee vain tarkastuksen että onko liittymäkohdat varatulla tieosalla.

Reply:
Totta tuokin.
Yleisesti: Tuollaisen lisäyksen pointti olisi siis valottaa sitä, miksi funktio ylipäänsä on olemassa. Tuo funktio on sen verran spesifi, että sitä tuskin tullaan käyttämään muuhun tarkoitukseen; näkisi saman tien funktion olemassaolon syyn.